web-client/src/components/figma/BlockStyles.tsx

```diff
@@ -78,4 +158,12 @@ export {
   LogoWrapper,
   OrDivider,
   StepWrapper,
+  SettingsWrapper,
```

Comment:
SettingsWrapper is good because I know what the settings wrapper is.
What is the "TopPanel"?